CondTools/Ecal/interface/EcalTPGFineGrainTowerfromFile.h

```diff
@@ -0,0 +1,58 @@
+#ifndef ECAL_TPGFINEGRAINTOWER_H
+#define ECAL_TPGFINEGRAINTOWER_H
+
+#include <vector>
```

Comment:
@depasse - it looks like only a few of these includes are needed in this header. Please remove the unneeded ones. Thanks